apns/analysis/apns2_ecut_qespresso.py

```diff
@@ -5,7 +5,7 @@ def is_outdir(folder: str):
     b = b and os.path.exists(os.path.join(folder, "pwscf.in"))
     return b
 
-def collect_jobs(folder: str):
+def collect(folder: str):
```

Comment:
thanks but presently I dont have plan to do this, because I think it is a good way to avoid name conflict by using something like wild card to import modules